### ISSUE TITLE
ld: library not found for -lsharpyuv

### ISSUE THREAD

**Issue by arjendevos** [discord/lilliput#138]
I've got the following error when trying to use lilliput 

```
/opt/homebrew/Cellar/go@1.20/1.20.10/libexec/pkg/tool/darwin_arm64/link: running c++ failed: exit status 1
ld: library not found for -lsharpyuv
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by arjendevos** [comment]
Go mod version:
```
	github.com/discord/lilliput v0.0.0-20230914023940-f3c639907ce4
```

**Comment by hroland** [comment]
+1

**Comment by skidder** [comment]
Resolved by #192
